packages/library-base/src/commonMain/kotlin/io/realm/kotlin/log/LogCategory.kt

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.kotlin.log
+
+/*
+ * Core does not expose the different categories in compile time.
```

Comment:
I think we should move all the internal stuff to an internal package/class/file. It confuses users if they end up here.